scripts/asm-installer/tests/run_migration_suite_citadel

```diff
@@ -57,6 +57,12 @@ main() {
   verify_demo_app "${GATEWAY}"
 
   # Test starts here
+
+  # Send persistent traffic to demo app to test any downtime.
+  echo "Start sending traffic to demo app"
+  send_persistent_traffic "${GATEWAY}" &
+  local SEND_TRAFFIC_CMD_PID="${!}"
```

Comment:
It probably won't happen, but if for some reason we fail to fork then this could be empty or -1. Later on you kill the pid. Bad things happen if you try to kill pid -1, especially if you happen to be root. Let's check the value of this immediately after forking to make sure the PID exists before continuing.

I _believe_ that the way you're using ps -p makes this safe, but 1) someone might change that later and forget about this, and 2) if we fail to fork then we probably don't want to continue the test anyways.

Reply:
Done.